assets/case/identifier-lookup.js

```diff
@@ -0,0 +1,53 @@
+/* global $ */
+
+window.addEventListener('ajaxload', function () {
+  const lookupElements = $('.identify-lookup')
```

Comment:
This should probably be changed to `identification-lookup` (also in the html).

Reply:
Changed to `identification-lookup`.